### ISSUE TITLE
Inform users who have different emails on their "submitter" vs user profile that further emails will go to the User profile email

### ISSUE THREAD

**Issue by abroddrick** [cisagov/manage.get.gov#2156]
### Issue description

We are collapsing submitter and creator as a concept which means we have to choose one or the other to save if there is a different between the two. Since the creator is associated with the User and can be updated via Your Contact anyways, we will be saving the values held in the "Creator" (User object and related Contact). That said we should be considerate of our users and inform them that further communication will be sent to whichever email is associated with the Creator contact. This email only needs to be sent if the submitter/creator email is different 

For further context emails are only sent to submitter for domain requests.

Refinement: notify all users who meet this criteria or just ones with active domain requests?

### Acceptance criteria

- [ ] write a script for finding all users who submitter email is different then the creator email on domain requests and then sending an email to them
- [ ] create a dev ticket for running this on stable

### Additional context

we have another migration script for sending emails, copy the -s parameter setup it uses to prevent accidental email sending.

refinement question: when should this be sent
Refinement: if we decide to not mock this as a "won't do", do we time this right as the user profile is getting turned on? Maybe it's better to send this when the user profile gets turned on.

[Slack discussion](https://cisa-corp.slack.com/archives/C05BGB4L5NF/p1718060960395159) for further clarifications.

### Links to other issues

- blocked by: #1807 
- blocked by #2047 
- blocked by #2167 (content for email)
- blocked by #2151 
- blocked by #2180 
- blocks ticket for running this script (make ticket if this feature is kept through **refinement**)

**Comment by vickyszuchin** [comment]
"refinement" tag added: @PaulKuykendall @abroddrick, to create supporting tickets for this ticket.

**Comment by vickyszuchin** [comment]
Per Alysia, move this ticket out of sprint 47 to sprint 48. There is not enough time to fully address this issue in the sprint given the lateness in resolving the blocker.

**Comment by abroddrick** [comment]
We have decided not to email users about this change
